.github/dependabot.yml

```diff
@@ -9,3 +9,6 @@ updates:
   directory: "/"
   schedule:
     interval: daily
+- name: Update Nix Flake Lock
```

Comment:
These two lines:

![image](https://github.com/fedimint/fedimint/assets/9209/58a04b3c-ece8-4028-a6a9-07f730771298)

;)

You're looking for `.github/workflows/ci-nix.yml`

Reply:
Actually, we probably want another workflow file, like `daily.yml`, but maybe let's do it once a week.

Reply:
made the changes.